### DISCUSSION TITLE
Extend model to retrieve other property

### DISCUSSION THREAD

**Discussion by tarikrital** [swellstores/community#495] [Help] [answered]
Welcome,

I'm working on listing products on the basket cart using swell-node, but I need to get customer name and I didn't find in Swell documentation a way to fix this!

![image](https://github.com/swellstores/community/assets/52907282/2ba36d2b-bde2-43c1-af14-1ec67b2b9300)

I don't see the new attribute in the console log of the returned product!

![image](https://github.com/swellstores/community/assets/52907282/9d343601-7f11-44b1-989a-8d42cea641f9)

**Comment by cheningD** [comment, marked as answer]
> but I need to get customer name 
do you mean `vendor_name`

First I would try expanding the query to include the fields you want: https://developers.swell.is/backend-api/querying/including#expanding

If that fails, the second thing i would try is list all vendors and make a vendor_id to vendor map

**Reply by tarikrital** [reply]
>do you mean vendor_name

Correct, I just mentioned the Model name.

**Reply by tarikrital** [reply]
Thank you, @cheningD.

![image](https://github.com/swellstores/community/assets/52907282/3cbee9e5-2080-4316-9509-1c9cc2201658)
